vite.config.js

```diff
@@ -5,6 +5,7 @@ import { svelte } from '@sveltejs/vite-plugin-svelte'
 export default defineConfig({
   plugins: [svelte()],
   server: {
+    port: 3000,
```

Comment:
This may confuse users, please remove.

Reply:
Using the port 3000 used to be default behavior in vite so I assumed the project was build with this in mind since this error gets spammed in the console if you use another one:
![image](https://user-images.githubusercontent.com/46646129/194405443-456e1b21-7b03-4cd9-ae5a-0eea1e2ce41e.png)

Reply:
yep this makes more sense! thanks for clarifying.